dev.py

```diff
@@ -279,6 +279,8 @@ def main(argv):
         os.execv(sys.executable, [sys.executable] + cmd)
         sys.exit(0)
 
+    test, version, mod_path = get_project_info()
+
```

Comment:
This was previously under if/else statement so didn't run when we used it with the building option. Now, this is fixed here.